sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/ApiManagementManagementTestBase.cs

```diff
@@ -25,17 +27,26 @@ protected ApiManagementManagementTestBase(bool isAsync)
         }
 
         [SetUp]
-        public void CreateCommonClient()
+        public async Task CreateCommonClient()
         {
             Client = GetArmClient();
+            DefaultSubscription = await Client.GetDefaultSubscriptionAsync().ConfigureAwait(false);
         }
 
-        protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionResource subscription, string rgNamePrefix, AzureLocation location)
+        protected async Task<ResourceGroupResource> CreateResourceGroupAsync()
         {
-            string rgName = Recording.GenerateAssetName(rgNamePrefix);
-            ResourceGroupData input = new ResourceGroupData(location);
-            var lro = await subscription.GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Completed, rgName, input);
-            return lro.Value;
+            var resourceGroupName = Recording.GenerateAssetName("testRG-");
```

Comment:
Lets make sure we avoid recording dependency set up.  @fengzhou-msft did we ever create a common pattern for this or is that still TBD?

Reply:
In `ManagementRecordedTestBase`, we've added [sanitizers](https://github.com/Azure/azure-sdk-for-net/blob/9b79912deb8c80851eb90ff0ded0dd936b0db6e0/common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs#L74-L104) for api-version in common ResourceManager operations. The resource group operations will be recorded but api-version is sanitized to `**`.

Reply:
I see that for [ResourceGroups](https://github.com/Azure/azure-sdk-for-net/pull/34105/files#diff-aa7132e4bb0ada0424a0162e56b32dcf21cba81e5d5b5746e06249713e758729R60) here, but should we do the same for [Subscriptions](https://github.com/Azure/azure-sdk-for-net/pull/34105/files#diff-aa7132e4bb0ada0424a0162e56b32dcf21cba81e5d5b5746e06249713e758729R4)?

Reply:
Looks like we had this comment addressed. Mark as resolved.